### PR TITLE
Revamp Code of Conduct

### PR DESCRIPTION
**Pull request by ramonh**
This adds:
- The tried and true [Berlin code of conduct](http://berlincodeofconduct.org/)
- A rake task to test the site locally
- A navbar link to the CoC

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -20,3 +20,7 @@ end
 task :jekyll do
   system "bundle exec jekyll build"
 end
+
+task :server do
```
Comment (helmuthb) [inline Rakefile]: I suppose you wanted to name the task "serve", not "server"?

File: cod.md [hunk 2]
```diff
@@ -6,8 +6,69 @@ isStaticPost: true
 ---
 
 DevFest Vienna is dedicated to being open to everyone and making the conference a safe place for all. We do not discriminate based on ethnic background, religion, gender, sexuality, body shape, or disability. We do not tolerate any harassment.
+Actions stronger than a warning will be taken at the discretion of two or more members of the DevFest organizer team.
+
+False reports will be treated harshly. Personal disputes should be handled privately, unless actual harassment occurs.
+
+_DevFest Vienna reserves the right to remove any participant, sponsor or attendee at any time for any reason._
+
+This policy covers all organized events for the DevFest Vienna, including social events.
```
Comment (helmuthb) [inline cod.md]: I think this sentence is now appearing twice - here and at the bottom, or?

File: Rakefile (hunk 1 above)
Comment (ramonh) [reply Rakefile]: I'm not picky :) I meant to write "server" like in Rails. Do you have a preference?

File: cod.md (hunk 2 above)
Comment (ramonh) [reply cod.md]: I don't think so:
https://github.com/DevFestVienna/devfest-at-site/blob/54704ded44fe66a2fc4cd2a74e12cfc593a133a1/cod.md

Comment (helmuthb) [reply cod.md]: https://github.com/DevFestVienna/devfest-at-site/blob/a1f4d493513e7211319441a5841d9c254ee24be5/cod.md

This is the one which is referenced in the PR - when clicking on "View File" on the right.

Let me check, I'll make a merge on my Desktop and see what's happening :slightly_smiling_face:

Comment (helmuthb) [reply cod.md]: Yes, if I follow the instructions the text appears then twice.

File: cod.md [hunk 3]
```diff
@@ -16,18 +77,27 @@ Alternate methods of reporting issues include:
...
+
+
+License and attribution
+-----------------------
+
+This is based on the [Berlin Code of Conduct](http://berlincodeofconduct.org/).
+
+Berlin Code of Conduct is distributed under a [Creative Commons Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)](https://creativecommons.org/licenses/by-sa/4.0/) license. It is based on the [pdx.rb Code of Conduct](https://pdxruby.org/CONDUCT).
 
 This policy covers all organized events for the DevFest Vienna, including social events.
```
Comment (helmuthb) [inline cod.md]: This is the second time this line appears.

File: cod.md (hunk 2 above)
Comment (ramonh) [reply cod.md]: Huh! Super odd. I'll remove it locally, just in case

File: cod.md (hunk 3 above)
Comment (ramonh) [reply cod.md]: Ahhh gotcha, thank you! I was looking for the wrong line, my bad

**Pull request merged**